apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.stories.tsx

```diff
@@ -7,6 +8,20 @@ import PoolCandidatesTable from "./PoolCandidatesTable";
 
 const poolCandidateData = fakePoolCandidates();
 
+const mockPoolCandidatesWithSkillCount = poolCandidateData.map(
+  (poolCandidate) => {
+    const skillCount = faker.datatype.number({
+      min: 0,
+      max: 10,
+    });
```

Comment:
Won't this result in a random skillCount number for each Chromatic run resulting in diffs every time?
My suggestion would be the same number for all for simplicity, or array index number = skillCount maybe since the array will always be the same.

Reply:
Hmm, that is a good point 🤔 I thought that the seed ensured the same number every time since [we are using it elsewhere](https://github.com/search?q=repo%3AGCTC-NTGC%2Fgc-digital-talent%20faker.datatype.number&type=code) and it seems stable.

Reply:
I think that requires this line to be called first somewhere in the file

https://github.com/GCTC-NTGC/gc-digital-talent/blob/53cdfcd3e0898ccdc25eb88b2ccc1531dc587bf3/packages/fake-data/src/fakePoolAdvertisements.ts#L69

Reply:
Hm, guess you don't need to declare seed 0 in the story file. Locally it matches with the web version. 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/f078fcfb-4e64-4a50-bb03-053f678e6419)